packages/mock/src/faker/getters/scalar.ts

```diff
@@ -137,7 +137,7 @@ export const getMockScalar = ({
     case 'number':
     case 'integer': {
       let value = getNullable(
-        `faker.number.int({min: ${item.minimum}, max: ${item.maximum}})`,
+        `faker.number.int({min: ${item.minimum}, max: ${item.maximum}, multipleOf: ${item.multipleOf}})`,
```

Comment:
How about making `multipleOf` available only for faker v9?

Reply:
Sounds good to me. Since this is merged and closed, is it okay if I do it in #2141 ?

Reply:
@SampsaKaskela 
Thank you. This is a different issue from #2142, so please create a new one.